Contest_API.md

```diff
@@ -1451,20 +1455,22 @@ The following endpoints are associated with state:
-| frozen           | TIME ? | Time when the scoreboard was frozen, or `null` if the scoreboard has not been frozen. Required iff `scoreboard_freeze_duration` is present in the [contest](#contests) endpoint.
-| ended            | TIME ? | Time when the contest ended, or `null` if the contest has not ended. Must not be set if started is `null`.
-| thawed           | TIME ? | Time when the scoreboard was thawed (that is, unfrozen again), or `null` if the scoreboard has not been thawed. Required iff `scoreboard_freeze_duration` is present in the [contest](#contests) endpoint. Must not be set if frozen is `null`.
-| finalized        | TIME ? | Time when the results were finalized, or `null` if results have not been finalized. Must not be set if ended is `null`.
-| end\_of\_updates | TIME ? | Time after last update to the contest occurred, or `null` if more updates are still to come. Setting this to non-`null` must be the very last change in the contest.
+| Name               | Type                | Description
+| :----------------- | :------------------ | :----------
+| started            | TIME ?              | Time when the contest actually started, or `null` if the contest has not started yet. When set, this time must be equal to the [contest](#contests) `start_time`.
+| frozen             | TIME ?              | Time when the scoreboard was frozen, or `null` if the scoreboard has not been frozen. Required iff `scoreboard_freeze_duration` is present in the [contest](#contests) endpoint.
+| ended              | TIME ?              | Time when the contest ended, or `null` if the contest has not ended. Must not be set if started is `null`.
+| thawed             | TIME ?              | Time when the scoreboard was thawed (that is, unfrozen again), or `null` if the scoreboard has not been thawed. Required iff `scoreboard_freeze_duration` is present in the [contest](#contests) endpoint. Must not be set if frozen is `null`.
+| finalized          | TIME ?              | Time when the results were finalized, or `null` if results have not been finalized. Must not be set if ended is `null`.
+| end\_of\_updates   | TIME ?              | Time after last update to the contest occurred, or `null` if more updates are still to come. Setting this to non-`null` must be the very last change in the contest.
+| removed\_intervals | array of INTERVAL ? | Array of [removed time intervals](ccs_system_requirements#removing-time-intervals).
 
 These state changes must occur in the order listed in the table above,
 as far as they do occur, except that `thawed` and `finalized` may occur
-in any order. For example, the contest may never be frozen and hence not
-thawed either, or, it may be finalized before it is thawed. That, is the
-following sequence of inequalities must hold:
+in any order. `removed_intervals` is not a state change, and so is not affected
+by this requirement. For example, the contest may never be frozen and hence not 
```

Comment:
This added sentences makes the flow of the example afterwards a bit worse, but I don't directly see a good way to fix that.

Reply:
Yes, I very much agree. I also did not see a good way :(.